packages/web-app-files/src/services/folder/loaderSpace.ts

```diff
@@ -33,10 +44,13 @@ export class FolderLoaderSpace implements FolderLoader {
   }
 
   public getTask(context: TaskContext): FolderLoaderTask {
-    const { router, clientService, resourcesStore, userStore } = context
-    const { webdav } = clientService
+    const { router, clientService, resourcesStore, userStore, spacesStore, sharesStore } = context
+    const { webdav, graphAuthenticated: graphClient } = clientService
```

Comment:
Didn't we have the issue of the different clients losing the access token reactivity when destructuring the clientService? 🤔

Reply:
This function gets called every time the loader performs, so the token would need to expire within the milliseconds between here and the function call 😅